src/Butler/Auth.hs

```diff
@@ -24,3 +25,19 @@ htmlMain xfiles title body = do
 
         with body_ [class_ "font-mono cursor-default bg-stone-100 h-screen"] do
             body
+
```

Comment:
This is stored in the Butler.Auth module however I'm wondering if that should be hosted in another one ?

Reply:
This is not ideal indeed, I'm open to suggestions.

Reply:
In Butler.Display perhaps ?